.github.com/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,84 @@
+
```

Comment:
.github.com directory is an artefact, you don't need it, it doesn't affect anything